code/modules/mob/living/carbon/human/life.dm

```diff
@@ -485,6 +485,8 @@
 	if(on_fire)
 		var/thermal_protection = get_thermal_protection()
 
+		if(wear_suit && wear_suit.flags & AUTOEXTINGUISH)
+			wear_suit.auto_extinguish(src)

```

Comment:
That'd mean that, like with the plasmamen 'igniting without their suit', having nobreathe will cure them of this.

Having nobreathe currently means Plasmamen can chill w/o their suit breathing oxygen and never ignite because ignition isn't handled based on atmosphere they're exposed to, but atmosphere they breathe-- and if they don't breathe... :confused: